examples/complete/replication.tf

```diff
@@ -1,13 +1,28 @@
 locals {
   replication_enabled = length(var.s3_replication_rules) > 0
-  extra_rule = local.replication_enabled ? {
-    id                 = "replication-test-explicit-bucket"
-    status             = "Enabled"
-    prefix             = "/extra"
-    priority           = 5
-    destination_bucket = module.s3_bucket_replication_target_extra[0].bucket_arn
-  } : null
-  s3_replication_rules = local.replication_enabled ? concat(var.s3_replication_rules, [local.extra_rule]) : null
+  replication_rules = local.replication_enabled ? [
+    {
+      id                 = "replication-test-explicit-bucket"
+      status             = "Enabled"
+      prefix             = "/extra"
+      priority           = 5
+      destination_bucket = module.s3_bucket_replication_target_extra[0].bucket_arn
+      destination        = null
+    },
+    {
+      id       = "replication-test-metrics"
+      status   = "Enabled"
+      prefix   = "/with-metrics"
+      priority = 10
+      destination = {
+        account_id = local.account_id
+        metrics = {
+          status = "Enabled"
+        }
+      }
+    }
+  ] : null
+  s3_replication_rules = local.replication_enabled ? local.replication_rules : null
```

Comment:
How come var.s3_replication_rules is no longer used ?

Reply:
that's my temp mess in tests nvm :)